pywb/static/wombat.js

```diff
@@ -78,6 +78,10 @@ var _WBWombat = function($wbwindow, wbinfo) {
     var wb_setAttribute = $wbwindow.Element.prototype.setAttribute;
     var wb_getAttribute = $wbwindow.Element.prototype.getAttribute;
     var wb_funToString = Function.prototype.toString;
+    var WBPreserWorker;
+    var wbSheetMediaQChecker;
+    var wbUsePresWorker = $wbwindow.Worker != null &&
+        (wbinfo.is_live || wbinfo.top_url.indexOf('/record/') !== -1);
```

Comment:
I think we shouldn't need the `/record/` check, `is_live` should be sufficient..

Reply:
Right you are. 
`is_live` is sufficient, changing